.github/workflows/devcontainer.yml

```
@@ -18,26 +36,38 @@ jobs:
           docker buildx build . \
```

```suggestion
          echo "Building a development image"
          docker buildx build . \
```
maybe we can add some annotations for run steps.

---

Good point. I created an issue #501 to add proper names, maybe we can do it there?

---

Got it.